types/index.d.ts

```
@@ -1046,142 +1003,200 @@ export declare function assert(received: any): Assertion;
  */
 export declare function is(received: any): Assertion;
 export type MethodNames<T> = {
-  [K in keyof T]: T[K] extends Function ? K : never;
+	[K in keyof T]: T[K] extends Function ? K : never;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)